docs/molgenis/dev_graphql.md

```diff
@@ -1,3 +1,70 @@
 # GraphQL in MOLGENIS
 
-TODO, describe basics. Meanwhile, please use self-documenting GraphQL IDE inside MOLGENIS.
\ No newline at end of file
+TODO, describe basics. Meanwhile, please use self-documenting GraphQL IDE inside MOLGENIS.
+
+# Getting started 
+
+## GraphQL API Playground
+
+In the top side of the menu you see a link to _GraphQL_ or _GraphQL API_
+this will open up a GraphQL playground for the selected database. <br>
+So that means, given the following url ```https://emx2.dev.molgenis.org/apps/central/#/ ``` <br>
+**apps** is the selected database. <br>
```

Comment:
this is incorrect , 'https://emx2.dev.molgenis.org/apps/central/#/' has no schema ( therefore it uses the system schema )